canonical/charmhub.io
# Rewrite discourse url to juju

## Done
- rewrite discourse topic url to juju.is
- refactor `get_docs`
## How to QA
- Go to https://charmhub-io-1648.demos.haus/docs?q={search-term}&page={page}
     - if page is 1, confirm that only 5 topics are returned in the list
     - confirm that each topic has an array of urls that matches tags (olm, sdk or dev)
- Go to https://charmhub-io-1648.demos.haus/docs/all-topics?q={search-term}&page={page}
    - confirm that each topic has an array of urls that matches tags (olm, sdk, dev)

## Issue / Card https://warthogs.atlassian.net/browse/WD-5270
